### PR TITLE
Restructuring of nomenclature for gases

### PR DESCRIPTION
**Pull request by sebwehrle**
Resolves some issues with conflicting names

### CODE REVIEW

**Review by danielhuppmann — APPROVED** [review]
Fine for me, but AEA is the more relevant party to comment...

File: definitions/variable/tag_final_energy.yaml [hunk 1]
```diff
@@ -18,23 +18,29 @@
...
+        description: Methane from biogenic sources
+    - Gas|Methane|Synthetic:
+        description: Synthesized methane
+    - Gas|Methane|Synthetic|Renewable Sources:
+        description: Synthesized methane from renewable sources
+    - Gas|Methane|Synthetic|Other Sources:
+        description: Synthesized methane from non-renewable sources
     - Gas|Biogas:
         description: Gas from biogenic sources including landfill gas, sewer gas, and 
           biogas (lower energy content than methane)
```
Comment (MartinBaumannAEA) [inline definitions/variable/tag_final_energy.yaml:41]: better: lower methane content than natural gas

File: definitions/variable/tag_electricity.yaml [hunk 2]
```diff
@@ -10,21 +10,30 @@
...
+    - Gas|Methane|Biogenic:
+        description: Methane from biogenic sources
+    - Gas|Methane|Synthetic:
+        description: Synthesized methane
+    - Gas|Methane|Synthetic|Renewable Sources:
+        description: Synthesized methane from renewable sources
+    - Gas|Methane|Synthetic|Other Sources:
+        description: Synthesized methane from non-renewable sources
     - Gas|Biogas:
-        description: Gas from biogenic sources (lower energy content than methane)
```
Comment (MartinBaumannAEA) [inline definitions/variable/tag_electricity.yaml:27]: better: lower methane content than natural gas

**Comment by MartinBaumannAEA** [comment]
Proposal is ok, just two comments on the description of biogas

File: definitions/variable/tag_final_energy.yaml (hunk 1 above)
Comment (sebwehrle) [reply definitions/variable/tag_final_energy.yaml]: implemented in new commit

File: definitions/variable/tag_electricity.yaml (hunk 2 above)
Comment (sebwehrle) [reply definitions/variable/tag_electricity.yaml:27]: implemented in new commit

**Review by leggler — APPROVED** [review]
thanks for your PR -approved. Only one note: In tag_electricity.yaml in the description adding the two words "electricity from..." (antracit, gas etc..) might add some clarity that we are talking about transformation input, not final use... however, this is only a minor comment.

**Comment by danielhuppmann** [comment]
> In tag_electricity.yaml in the description adding the two words "electricity from..." (antracit, gas etc..) might add some clarity that we are talking about transformation input, not final use...

This is already included in the description of the variable, see

https://github.com/netzero2040/netzero2040/blob/ca4c9ebf2eeebe5e6ce5c0842536afcf79912893/definitions/variable/energy.yaml#L6

**Pull request merged**